t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -365,9 +365,8 @@ sub wait_for_element {
                 && (!defined $expected_is_displayed || $elements[0]->is_displayed == $expected_is_displayed))
             {
                 $element = $elements[0];
```

Comment:
But this line - "which is dead code unless the test is slow enough"¹ - is still conditionally executed.
¹ I assume this sentence refers to this block of code.

Reply:
The gap in coverage is on the `return 0` (if you're looking at older reports like the one in the ticket it may be misleading since the line number changed by 2 in master)

Reply:
If we never hit the `if` the test bails, so that could only be a gap if the test has already failed.